src/types/markers.js

```diff
@@ -44,6 +44,11 @@ export type PaintProfilerMarkerTracing = {
   interval: 'start' | 'end',
 };
 
+export type ArbitraryEventTracing = {|
+  type: 'tracing',
+  category: string,
+|};
```

Comment:
nit: can you make the object readonly ? Using `+` before each property, or `$ReadOnly<...>` around the object.

Maybe we should have a `type $Strict<O> = $ReadOnly<$Exact<O>>` and use it everywhere :) (`$Exact` is like `{| .. |}`)